tensorflow/compiler/xla/debug_options_flags.cc

```diff
@@ -70,20 +70,16 @@ DebugOptions DefaultDebugOptionsIgnoringFlags() {
 
   opts.set_xla_gpu_enable_cudnn_frontend(true);
 
-<<<<<<< HEAD
   // TODO(b/241801928): Remove this flag and legacy cublas support once cublasLt
   // is fully supported
 #if GOOGLE_CUDA
   opts.set_xla_gpu_enable_cublaslt(true);
 #else
   opts.set_xla_gpu_enable_cublaslt(false);
 #endif
-=======
-  opts.set_xla_gpu_enable_cublaslt(false);
```

Comment:
Let's just revert to what upstream has here (just set to false with no #if's) 
Once they turn it back on we can redo the ifs (or if we have hipblasLT support then, maybe we keep it)

Reply:
Done